.github/workflows/release.yml

```diff
@@ -48,6 +50,7 @@ jobs:
       - uses: actions/setup-node@v6
         with:
           node-version: 22
```

Comment:
Node 22 ships with npm 10.9.4. Ref: https://nodejs.org/en/about/previous-releases But Trusted Publishing requires npm 11.5.1 or higher.

So I think this workflow would also require updating npm with `npm install npm@11 -g`.

Reply:
second issue i mentioned is fixed. i had accidently remvoed contents: write from the publish step so that is what the other action was failing

https://github.com/acalcutt/node-pre-gyp/actions/runs/20175726794/job/57923175131

Reply:
Looks good! Updating to Node 24 will get us npm 11.6.2 which should work.